### ISSUE TITLE
[Appium 1.5] 'BadParametersError' error raised when trying to set network mode

### ISSUE THREAD

**Issue by JaniJegoroff** [appium/appium#6290]
## The problem

I'm using `set_network_connection` command successfully (Ruby bindings) with Appium server 1.4.16 to set network mode offline/online on Android. Looks like this is not working anymore with Appium server 1.5. I did not find any notes mentioned regarding to this in release notes. Anyone experiencing the same issue?

## Environment

* Appium version (or git revision) that exhibits the issue: 1.5.0
* Last Appium version that did not exhibit the issue (if applicable): 1.4.16
* Desktop OS/version used to run Appium: Mac OS X El Capitan 10.11.3
* Node.js version (unless using Appium.app|exe): v5.1.1
* Mobile platform/version under test: Android
* Real device or emulator/simulator: Real device
* Appium CLI or Appium.app|exe: CLI
* Other: ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-darwin14.0]

## Details

```
Appium 1.4.16
Janis-MacBook-Air:android_tests janijegoroff$ arc
[1] pry(main)> get_network_connection
6
[2] pry(main)> set_network_connection 1
1
[3] pry(main)> set_network_connection 6
6
[4] pry(main)>
```
```
Appium 1.5.0
Janis-MacBook-Air:android_tests janijegoroff$ arc
[1] pry(main)> get_network_connection
6
[2] pry(main)> set_network_connection 1
Selenium::WebDriver::Error::WebDriverError: unexpected response, code=400, content-type="text/html"
Parameters were incorrect. We wanted {"unwrap":"parameters","required":["type"]} and you sent []
from /Users/janijegoroff/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/selenium-webdriver-2.50.0/lib/selenium/webdriver/remote/http/common.rb:85:in `create_response'
[3] pry(main)>
```

## Link to Appium logs

https://gist.github.com/JaniJegoroff/42b2e1b6ea3e98ba455f

## Code To Reproduce Issue [ Good To Have ]

1. Fork https://github.com/appium/ruby_lib
2. Connect Android device
3. Start Appium server
4. Navigate to `ruby_lib/android_tests` directory
5. Execute `bundle`
6. Execute `arc`
7. Execute `get_network_connection`
8. Execute `set_network_connection 1`

**Comment by imurchie** [comment]
Looking into this.

**Comment by imurchie** [comment]
Fixed in `mobile-json-wire-protocol@1.3.3`. If you're running Appium from source, you can update.
